Doc changes for stack management and grouped feature privileges (#80486)
elastic/kibana: docs/management/managing-beats.asciidoc

enrollment and configuration process step by step the first time you use the
Central Management UI.

[float]
=== Required permissions

You must have the `beats_admin` role assigned to use **{beats} Central Management**

> You can assign this role in *Stack Management > Security > Users*.

Reviewer: ```suggestion
To assign the role, open the menu, then click *Stack Management > Users*.
```
